bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/EchoHandler.java

```diff
@@ -1184,7 +1145,7 @@ public void updateState(AccountHandler accountHandler, @Nullable Device device,
             }
 
         } catch (Exception e) {
-            this.logger.debug("Handle updateState {} failed: {}", this.getThing().getUID(), e);
+            this.logger.debug("Handle updateState {} failed: {}", this.getThing().getUID(), e.getMessage());
```

Comment:
I kept the exception for the logger but I would like to not touch the catch-thing. Without checking what is really possible and expected here it might get worse. IMO we should raise the severity of catching `Exception` and `Throwable` to 1, so that the build fails